beta-documentation/returns-api-PROJECT-2776/returns.json

```diff
@@ -131,7 +131,7 @@
         "tags": [
           "Returns"
         ],
-        "description": "Applies status updates to a return a collection of returns",
+        "description": "Applies status updates to a collection of returns",
```

Comment:
💯 

Even though this was a typo, it's a good point that we need to be cautious about returning returns 😆